components/comparison.py

```diff
@@ -259,7 +259,7 @@ def get_patch_df(df, player_df, patch):
         patch_df = player_df[player_df.patch == patch]
     elif patch == Graph.last_16.value:
         hidden_query = {} if hidden_features else dict(public=True)
-        qs = set(TourneyResult.objects.filter(league=champ, **hidden_query).order_by("-date").values_list("date", flat=True)[:16])
+        qs = set(TourneyResult.objects.filter(league=legend, **hidden_query).order_by("-date").values_list("date", flat=True)[:16])
```

Comment:
This just makes the page less usable (unusable currently, since there's no legend tourneys, but less usable for the future, where you can't see historical data), revert.